sdk/src/main/java/com/hedera/hashgraph/sdk/FeeSchedule.java

```diff
@@ -0,0 +1,60 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import java.util.List;
+import java.util.ArrayList;
+
+import org.threeten.bp.Instant;
+
+public class FeeSchedule {
+    private List<TransactionFeeSchedule> transactionFeeSchedules;
+    private Instant expiryTime;
```

Comment:
Can you rename `expiryTime` to `expirationTime`; we've done that several times throughout the SDK for consistency.